src/uadk_cipher.c

```diff
@@ -27,13 +27,13 @@
 #define CTX_ASYNC_DEC	3
 #define CTX_NUM		4
 
-struct digest_engine {
+struct cipher_engine {
 	struct wd_ctx_config ctx_cfg;
 	struct wd_sched sched;
 	int pid;
 };
 
-static struct digest_engine engine;
+static struct cipher_engine engine;
```

Comment:
do not change cipher.c in this patch, we can change this later with cipher patch